app/src/components/map/InteractionLayer.jsx

```diff
@@ -78,7 +78,12 @@ const InteractionLayer = ({ baseStyles, interactionStyles }) => {
 
   useEffect(() => {
     if (!mapRef.current) return;
-
+    if (!mapRef.current.getPane("interactive")) {
```

Comment:
I suspect this block is about disabling the interactivity with the map when it isn't focused? Could we get a comment to just tell what/why this code is here?